standalone/test/integration_tests/api/integrationtest_scanner_api.cpp

```diff
@@ -92,9 +91,9 @@ class UserCallbacks
     EXPECT_START_REQUEST_CALL(*hw_mock, *config).WillOnce(OpenBarrier(&start_req_barrier));                            \
     EXPECT_NO_BLOCK_NO_THROW(start_future = driver->start(););                                                         \
     EXPECT_TRUE(start_future.valid());                                                                                 \
-    EXPECT_BARRIER_OPENS(start_req_barrier, DEFAULT_TIMEOUT) << "Start request not received";                          \
+    start_req_barrier.waitTillRelease(2s);                                                                             \
```

Comment:
Is there a specific reason for decreasing the timeout (was 3s before)? Keep in mind we still have #262 open. These failures might not have occured for a bit looser timeouts.

Reply:
the #262 runs against a different 1s timeout in a method that should not even take .1 seconds.
The whole test cases are usually done in far less that one second. I think high timeouts only resolve in costing us more time during development with the risk of covering other problems. If it wasn't for the UDP in those tests i would even pledge for timeouts of less than a second. I thought going down to 2s will reduce the time we wait for the CI in fail cases by 30% without really risking to cause problems due to to small timeouts.